kloppy/infra/serializers/tracking/epts/serializer.py

```diff
@@ -154,14 +149,9 @@ def deserialize(
             else None
```

Comment:
Here Orientation is the only property of meta_data that is being set here instead that on meta_data.py, which might not be ideal and might be preferred to set everything in meta_data.py. The reasons why I dindn't do that are:

1. It will still need to check in serializer.py if it's none, and if so try to get it from the first frame (which bt might not be a good idea as sometimes the first frame is garbage)
2. Since the `orientation = (...` would be called in two places it would be better to abstract it but didn't know where would be the right place for that method to be placed.

Reply:
When there is a way to retrieve it from the EPTS metadata we should use that, and otherwise just leave it None. Then we can add a OrientationEstimator or something that gets a couple of frames as input and returns the Orientation. That will make the estimating of the orientation more generic and can be removed from all other serializers too.